awx/main/tasks.py

```diff
@@ -1217,9 +1220,6 @@ def run(self, pk, **kwargs):
                     module_args = ansible_runner.utils.args2cmdline(
                         params.get('module_args'),
                     )
-                else:
-                    # otherwise, it's a playbook, so copy the project dir
-                    copy_tree(cwd, os.path.join(private_data_dir, 'project'))
```

Comment:
🎉

Reply:
Still too early for celebration?

This wasn't removed, just moved to the pre_run_hook so that it happens for every job run, not just isolated.